src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/TransferEvidenceTonnageViewModelMap.cs

```diff
@@ -47,10 +47,10 @@ public TransferEvidenceTonnageViewModel Map(TransferEvidenceNotesViewModelMapTra
                 if (source.TransferAllTonnage)
                 {
                     category.TotalReceived = source.Notes.SelectMany(n => n.EvidenceTonnageData)
```

Comment:
There should be a unit test to update for this? If not we should create a unit test to catch this bug.